fineract-provider/src/main/java/org/apache/fineract/portfolio/note/domain/Note.java

```diff
@@ -37,150 +38,46 @@
 import org.apache.fineract.portfolio.shareaccounts.domain.ShareAccount;
 
 @Entity
+@Getter
+@Builder
+@NoArgsConstructor
```

Comment:
do we need the public no-arg and all-args constructors if we plan to switch to Builder?

Reply:
Yes, the `@Builder` annotation requires a proper constructor for initializing the entity>

Reply:
my point is that we should not have public constructors, if we are switching to the builder pattern. Please make sure that constructors are having the proper access level.